src/qml/components/KeyboardHandler.qml

```diff
@@ -0,0 +1,107 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+import "../base"
+import "../theme"
+import "../components"
+
+
+Item {
+    id: keyboardHandler
+
+    property real keyboardHeight: 0
+    property var constKeyboardRectangle
+    property real constKeyboardYTranslate: 0
+    property var currentKeyboardRectangle
+    property bool keyboardOpen: false
+    onKeyboardOpenChanged: {
+        if (!Platform.isDesktop) {
```

Comment:
I don't think this check is needed. Just disable Connections {} for Desktop platforrms